README.rst

```diff
@@ -1,125 +1,82 @@
+If you are looking for a successor to ``epydoc`` after moving to Python 3,
+``pydoctor`` might be the right tool for your project as well.
 
-pydoctor puts a fair bit of effort into resolving imports and
+``pydoctor`` puts a fair bit of effort into resolving imports and
 computing inheritance hierarchies and, as it aims at documenting
-Twisted, knows about zope.interface's declaration API and can present
+Twisted, knows about ``zope.interface``'s declaration API and can present
 information about which classes implement which interface, and vice
 versa.
 
 .. contents:: Contents:
 
 
-Usage
------
+Simple Usage
+~~~~~~~~~~~~
 
 You can run pydoctor on your project like this::
 
     $ pydoctor --make-html --html-output=docs/api --add-package=src/mylib
 
 You can see the full list of options using ``pydoctor --help``.
 
+For more info, `Read The Docs <https://pydoctor.readthedocs.io/>`_.
+
 Markup
-------
+~~~~~~
 
 pydoctor currently supports the following markup languages in docstrings:
 
 `epytext`__ (default)
     The markup language of epydoc.
     Simple and compact.
 
-`reStructuredText`__
+`restructuredtext`__
     The markup language used by Sphinx.
     More expressive than epytext, but also slighly more complex and verbose.
 
-plain text
+plaintext
     Text without any markup.
 
 __ http://epydoc.sourceforge.net/manual-epytext.html
 __ https://docutils.sourceforge.io/rst.html
 
 You can select a different format using the ``--docformat`` option.
 
-Sphinx Integration
-------------------
-
-Sphinx object inventories can be used to create deep links between API
-documentation generated by pydoctor and by Sphinx.
-
-Linking to external API docs
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-To link to the Python standard library documentation, add this to your
-pydoctor command line::
-
-    --intersphinx=https://docs.python.org/3/objects.inv
-
-You can pass this option multiple times to also link to other libraries
-that provide a Sphinx objects inventory.
-
-Linking to your API docs
-~~~~~~~~~~~~~~~~~~~~~~~~
-
-pydoctor's HTML generator will also generate a Sphinx objects inventory
-using the following mapping:
-
-* packages, modules -> py:mod:
-* classes -> py:class:
-* functions -> py:func:
-* methods -> py:meth:
-* attributes -> py:attr:
-
-To use this mapping in Sphinx, configure the `intersphinx extension`__::
-
-    intersphinx_mapping = {
-        'pydoctor': ('http://domain.tld/api', None),
-    }
-
-__ https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html
-
-And use external references::
-
-    :py:func:`External API <pydoctor:pydoctor.model.Documentable.reparent>`
-
-    :py:mod:`pydoctor:pydoctor`
-    :py:mod:`pydoctor:pydoctor.model`
-    :py:func:`pydoctor:pydoctor.driver.getparser`
-    :py:class:`pydoctor:pydoctor.model.Documentable`
-    :py:meth:`pydoctor:pydoctor.model.Documentable.reparent`
-    :py:attr:`pydoctor:pydoctor.model.Documentable.kind`
-
 What's New?
------------
+~~~~~~~~~~~
```

Comment:
This is ok for now, but maybe is better to move it into a separate NEWS / Changelog / CHANGES file.